package.json

```diff
@@ -5,13 +5,17 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "prettier --write \"src/**/*.{js,scss,vue}\" && eslint --fix \"src/**/*.{js,vue}\"",
```

Comment:
Nice; as we've discussed before w.r.t Black, I'm all for uniform styles!

It might be possible to integrate prettier with the [existing `vue-cli-service lint` command](https://cli.vuejs.org/core-plugins/eslint.html#injected-commands) but happy to roll with this too.

Reply:
Looking into this.